events/elections/2020/templates/election-templates.md

```diff
@@ -0,0 +1,105 @@
+
+Resources:
+
+[1] Steering Committee - who sits on the committee and terms, their projects and meeting info  
+[2] Steering Committee Charter - this is a great read if you’re interested in running (or assessing for the best candidates!)
+[3] Election Process
+[4] Voters Guide! - Updated on a rolling basis. This guide will always have the latest information throughout the election cycle. The complete schedule of events and candidate bios will be housed here.
+
+On behalf of the election officers,
+
+Name (github), Name (github), etc
+
+If you have questions, please feel free to ask on the list; otherwise, you can reach the Election Officials at election@kubernetes.io.
+
+## Election Status Updates
+
+Post these weekly to kubernetes-dev
+
+$greeting,
+
+The Kubernetes Steering Commitee Election begins today/is underway! Here's what you need to know as we begin prep to send out the ballots.
+
+    Ballots are going out today via email. Expect a mail from "Kubernetes Election Officers (CIVS poll supervisor) <ci...@cs.cornell.edu>" with a subject like "Poll: Kubernetes Steering Committee Election 2020"
+        Email can be unreliable, so generally expect them this morning, if you haven't received one by the end of the day let us know, sending you a new ballot is easy for us.
+
+The candidates and links to their bios are all available here: https://git.k8s.io/community/events/elections/2020
+
+// Voting
+
+- You should check to see if you are in voters.md. Your github handle MUST BE IN THIS DOCUMENT to vote: https://github.com/kubernetes/community/blob/master/events/elections/2020/voters.md
+
+- If you feel you have made enough contributions to be able to vote but you are NOT in voters.md, you need to fill out this form by XXX: https://www.surveymonkey.com/r/k8s-sc-election-2019
+
+- If you are planning on running for Steering Committee please see this section of the documentation, you need to PR your biography into the repo by XXX: https://github.com/kubernetes/community/tree/master/events/elections/2019#candidacy-process
+
+
+// Schedule
+
+The next deadline is XXX , you have until then to complete your ballot. If you have any questions, let us know. 
+
+| September 11 | All candidate bios and voting exception forms due by 0000 UTC (5pm PST) |
+| September 18 | Election Begins via email ballots |
+| October 2    | Election Closes by 0000 UTC (5pm PST) |
+| October 3    | Announcement of Results at Community Meeting |
+
+
+// Resources
+
+Voters Guide - https://git.k8s.io/community/events/elections/2020
```

Comment:
nit: maybe we could make these bullet points so that they show up nicely in markdown?